src/app/app.component.ts

```diff
@@ -9,6 +9,6 @@ import { AuthClientService } from './services/auth.client.service';
 export class AppComponent {
   title = 'a0-with-ssr';
 
-  constructor(public auth: AuthClientService){}
-  
+  constructor(public auth: AuthService){}
+
```

Comment:
Reverted using a wrapper for simplicity. Even though you can still use a wrapper if you want to.